.github/workflows/checks.yml

```diff
@@ -149,7 +158,7 @@ jobs:
           chmod u+x darwinia
           sudo mv darwinia /usr/bin
       - name: Launch darwinia
-        run: darwinia --chain pangolin-dev --tmp --alice --tracing-api debug,trace &
+        run: darwinia --chain pangolin-dev --tmp --alice --tracing-api debug,trace --frontier-backend-type sql &
```

Comment:
It's better to keep the original task without `--frontier-backend-type sql`, because not all of the collator nodes will upgrade their fronter db in the short term.

Reply:
IMO, only RPC node will use this. Collator doesn't care about this.